writeimage.sh

```diff
@@ -13,16 +12,52 @@ function usage() {
     exit 1
 }
 
+function msg() {
+    echo "$(date) $1"
+}
+
+# use this function to unmount regardless of MAC OS or not
+function smartUnmount() {
+	msg "Unmounting $1"
+
+	# if MAC OS
+	if [ `uname` == "Darwin" ]; then
+	    diskutil unmountDisk $1
+	else # assuming Linux
+		# unmount sdcard
+    	umount $1* >/dev/null 2>&1
+	fi
+}
+
+# use this function to mount regardless of MAC OS or not
+function smartMount() {
```

Comment:
True.  No need for this function. I originally created this function to mount, and then found that instead of mounting with diskUtil I had to use hdiutil attach. This function could be removed.